src/styles/cardbuy.scss

```diff
@@ -0,0 +1,19 @@
+.card__buy {
+  box-sizing: border-box;
+  width: 166px;
```

Comment:
If we want button on the full width, better to set `width: 100%`, and for the button wrapper(tag a) same